# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1327966436 -3600
#      Tue Jan 31 00:33:56 2012 +0100
# Branch gui
# Node ID fdd5828e2214a24eb67479061e77c1a41f7ac195
# Parent  e7d9e6d8c6b549d73ad538550f4d31ac9d0cfb9d
Removed ImageViewer. Removed stylesheets from dock widgets.
* FilesDockWidget.cpp: Removed stylesheet.
* HistoryDockWidget.cpp: Removed stylesheet.
* WorkspaceView.cpp: Removed stylesheet.

diff --git a/gui/octave-gui.pro b/gui/octave-gui.pro
--- a/gui/octave-gui.pro
+++ b/gui/octave-gui.pro
@@ -75,17 +75,16 @@ QMAKE_CXXFLAGS      += $$INCFLAGS
 SOURCES +=\
     src/lexer/lexeroctavegui.cpp \
     src/MainWindow.cpp \
     src/WorkspaceView.cpp \
     src/HistoryDockWidget.cpp \
     src/FilesDockWidget.cpp \
     src/FileEditorMdiSubWindow.cpp \
     src/BrowserWidget.cpp \
-    src/ImageViewerMdiSubWindow.cpp \
     src/SettingsDialog.cpp \
     src/OctaveGUI.cpp \
     src/ResourceManager.cpp \
     src/CommandLineParser.cpp \
     src/backend/OctaveCallbackThread.cpp \
     src/backend/OctaveLink.cpp \
     src/backend/OctaveMainThread.cpp \
     src/backend/ReadlineAdapter.cpp \
@@ -98,17 +97,16 @@ SOURCES +=
 HEADERS += \
     src/lexer/lexeroctavegui.h \
     src/MainWindow.h \
     src/WorkspaceView.h \
     src/HistoryDockWidget.h \
     src/FilesDockWidget.h \
     src/FileEditorMdiSubWindow.h \
     src/BrowserWidget.h \
-    src/ImageViewerMdiSubWindow.h \
     src/SettingsDialog.h \
     src/ResourceManager.h \
     src/CommandLineParser.h \
     src/backend/OctaveCallbackThread.h \
     src/backend/OctaveLink.h \
     src/backend/OctaveMainThread.h \
     src/backend/ReadlineAdapter.h \
     src/WelcomeWizard.h
diff --git a/gui/src/FilesDockWidget.cpp b/gui/src/FilesDockWidget.cpp
--- a/gui/src/FilesDockWidget.cpp
+++ b/gui/src/FilesDockWidget.cpp
@@ -94,27 +94,16 @@ FilesDockWidget::FilesDockWidget (QWidge
 
   connect (m_currentDirectory, SIGNAL (returnPressed ()), this,
 	   SLOT (currentDirectoryEntered ()));
   QCompleter *
     completer = new QCompleter (m_fileSystemModel, this);
   m_currentDirectory->setCompleter (completer);
 
   connect (this, SIGNAL (visibilityChanged(bool)), this, SLOT(handleVisibilityChanged(bool)));
-
-  setStyleSheet(
-      " QDockWidget { "
-      "   border:1px solid rgba(200, 200, 200, 255); "
-      "   background-color: qlineargradient(spread:pad, "
-      "   x1:0.1, y1:0, x2:0.9, y2:1, "
-      "   stop:0.256444 rgba(215, 215, 255, 255), "
-      "   stop:0.550888 rgba(255, 255, 255, 255), "
-      "   stop:0.780424 rgba(215, 215, 255, 255)); "
-      " } "
-      );
 }
 
 void
 FilesDockWidget::itemDoubleClicked (const QModelIndex & index)
 {
   // Retrieve the file info associated with the model index.
   QFileInfo fileInfo = m_fileSystemModel->fileInfo (index);
 
diff --git a/gui/src/HistoryDockWidget.cpp b/gui/src/HistoryDockWidget.cpp
--- a/gui/src/HistoryDockWidget.cpp
+++ b/gui/src/HistoryDockWidget.cpp
@@ -44,27 +44,16 @@ HistoryDockWidget::construct ()
   layout->addWidget (m_filterLineEdit);
   layout->setMargin (2);
 
   widget ()->setLayout (layout);
 
   connect (m_filterLineEdit, SIGNAL (textEdited (QString)), &m_sortFilterProxyModel, SLOT (setFilterWildcard(QString)));
   connect (m_historyListView, SIGNAL (doubleClicked (QModelIndex)), this, SLOT (handleDoubleClick (QModelIndex)));
   connect (this, SIGNAL (visibilityChanged(bool)), this, SLOT(handleVisibilityChanged(bool)));
-
-  setStyleSheet(
-      " QDockWidget { "
-      "   border:1px solid rgba(200, 200, 200, 255); "
-      "   background-color: qlineargradient(spread:pad, "
-      "   x1:0.1, y1:0, x2:0.9, y2:1, "
-      "   stop:0.256444 rgba(215, 255, 215, 255), "
-      "   stop:0.550888 rgba(255, 255, 255, 255), "
-      "   stop:0.780424 rgba(215, 255, 215, 255)); "
-      " } "
-      );
 }
 
 void
 HistoryDockWidget::handleDoubleClick (QModelIndex modelIndex)
 {
   emit commandDoubleClicked (modelIndex.data().toString());
 }
 
diff --git a/gui/src/ImageViewerMdiSubWindow.cpp b/gui/src/ImageViewerMdiSubWindow.cpp
deleted file mode 100644
--- a/gui/src/ImageViewerMdiSubWindow.cpp
+++ /dev/null
@@ -1,43 +0,0 @@
-/* OctaveGUI - A graphical user interface for Octave
- * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
- * published by the Free Software Foundation, either version 3 of the
- * License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program. If not, see <http://www.gnu.org/licenses/>.
- */
-
-#include "ImageViewerMdiSubWindow.h"
-#include <QLabel>
-#include <QPixmap>
-#include <QScrollArea>
-
-ImageViewerMdiSubWindow::ImageViewerMdiSubWindow (QPixmap pixmap, QWidget * parent):QMdiSubWindow (parent),
-m_pixmap
-(pixmap)
-{
-  construct ();
-}
-
-void
-ImageViewerMdiSubWindow::construct ()
-{
-  QLabel *label = new QLabel ();
-  label->setBackgroundRole (QPalette::Base);
-  label->setSizePolicy (QSizePolicy::Ignored, QSizePolicy::Ignored);
-  label->setScaledContents (true);
-  label->setPixmap (m_pixmap);
-
-  QScrollArea *scrollArea = new QScrollArea (this);
-  scrollArea->setBackgroundRole (QPalette::Dark);
-  scrollArea->setWidget (label);
-  setWidget (scrollArea);
-}
diff --git a/gui/src/ImageViewerMdiSubWindow.h b/gui/src/ImageViewerMdiSubWindow.h
deleted file mode 100644
--- a/gui/src/ImageViewerMdiSubWindow.h
+++ /dev/null
@@ -1,33 +0,0 @@
-/* OctaveGUI - A graphical user interface for Octave
- * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
- * published by the Free Software Foundation, either version 3 of the
- * License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program. If not, see <http://www.gnu.org/licenses/>.
- */
-
-#ifndef IMAGEVIEWERMDISUBWINDOW_H
-#define IMAGEVIEWERMDISUBWINDOW_H
-
-#include <QMdiSubWindow>
-
-class ImageViewerMdiSubWindow:public QMdiSubWindow
-{
-public:
-  ImageViewerMdiSubWindow (QPixmap pixmap, QWidget * parent = 0);
-
-private:
-  void construct ();
-  QPixmap m_pixmap;
-};
-
-#endif // IMAGEVIEWERMDISUBWINDOW_H
diff --git a/gui/src/MainWindow.cpp b/gui/src/MainWindow.cpp
--- a/gui/src/MainWindow.cpp
+++ b/gui/src/MainWindow.cpp
@@ -19,17 +19,16 @@
 #include <QMenu>
 #include <QAction>
 #include <QSettings>
 #include <QDesktopServices>
 #include <QFileDialog>
 #include <QMessageBox>
 #include "MainWindow.h"
 #include "FileEditorMdiSubWindow.h"
-#include "ImageViewerMdiSubWindow.h"
 #include "SettingsDialog.h"
 
 #define VERSION_STRING "Octave GUI (0.8.8)"
 
 MainWindow::MainWindow (QWidget * parent):QMainWindow (parent)
 {
   construct ();
   OctaveLink::instance ()->launchOctave();
diff --git a/gui/src/WorkspaceView.cpp b/gui/src/WorkspaceView.cpp
--- a/gui/src/WorkspaceView.cpp
+++ b/gui/src/WorkspaceView.cpp
@@ -55,28 +55,16 @@ WorkspaceView::WorkspaceView (QWidget * 
   treeWidgetItem->setData (0, 0, QString (tr ("Hidden")));
   m_variablesTreeWidget->insertTopLevelItem (3, treeWidgetItem);
 
   m_variablesTreeWidget->expandAll ();
   m_variablesTreeWidget->setAlternatingRowColors (true);
   m_variablesTreeWidget->setAnimated (true);
 
   connect (this, SIGNAL (visibilityChanged(bool)), this, SLOT(handleVisibilityChanged(bool)));
-
-  setStyleSheet(
-      " QDockWidget { "
-      "   border:1px solid rgba(200, 200, 200, 255); "
-      "   background-color: qlineargradient(spread:pad, "
-      "   x1:0.1, y1:0, x2:0.9, y2:1, "
-      "   stop:0.256444 rgba(255, 215, 215, 255), "
-      "   stop:0.550888 rgba(255, 255, 255, 255), "
-      "   stop:0.780424 rgba(255, 215, 215, 255)); "
-      " } "
-      );
-
   connect (OctaveLink::instance(), SIGNAL (symbolTableChanged()), this, SLOT (fetchSymbolTable()));
 }
 
 void
 WorkspaceView::updateTreeEntry (QTreeWidgetItem * treeItem, SymbolRecord symbolRecord)
 {
   treeItem->setData (0, 0, QString (symbolRecord.name ().c_str ()));
   treeItem->setData (1, 0,
